www/log.css

```diff
@@ -171,7 +179,7 @@ td
 	text-align: right;
     vertical-align: top;
 	border-right: var(--divider);
-	padding-right: 0.5em;
+    padding-right: 0.5em;
```

Comment:
Your editor seems to be mixing tabs and spaces here.

Reply:
It looks like there are mixed tabs and spaces all over, so my editor got confused. Something to rectify in the future.